mods/ctf/ctf_map/maps_catalog.lua

```diff
@@ -133,25 +158,23 @@ minetest.register_chatcommand("maps", {
 		end
 
 		local player = minetest.get_player_by_name(name)
-		local idx
 
-		-- If arg. supplied, set idx to index of the matching map name
-		-- or path. Else, set to indices[name] or index of current map
 		if param then
-			idx = ctf_map.get_idx_and_map(param)
-		else
-			idx = (player and indices[name]) or ctf_map.map and ctf_map.map.idx or 1
+			indices[name] = ctf_map.get_idx_and_map(param)
+		end
+		if not player or not indices[name] then
```

Comment:
```suggestion
		if not player and not param or not indices[name] then
```

Otherwise it will always (even when a parameter is specified) show the currently played map in IRC.

Reply:
Done